browse/services/search/search_authors.py

```diff
@@ -8,37 +7,44 @@
 
 
 def is_affiliation(item: str)-> bool:
+    """Return true if a string contains an affiliation."""
     return item.startswith('(')
 
 
 def is_short(item: str)-> bool:
+    """Return true if the length of string is less than 4 characters long."""
```

Comment:
These comments seem redundant. Is this just to achieve a pylint score?

Reply:
pydocstyle checks complain (fail?) otherwise.

Reply:
Ah.